tensorrt_llm/_torch/models/modeling_utils.py

```
@@ -75,6 +75,24 @@ class MetaInitMode(TorchDispatchMode):
+    # allocator (mimalloc) retains the freed weight-shard-sized arena for
+    # process lifetime (measured 142 GiB/rank for a ~554 GB checkpoint at
+    # TP4 on GB300). Their results are placeholders that are overwritten
+    # after construction (dummy init or checkpoint load), so they are safe
+    # to skip on meta tensors. Do NOT add ops whose results are consumed
+    # (e.g. ops computing non-persistent buffers): the conservative
+    # MetaInitException fallback below still protects those.
+    deterministic_init_ops = {
+        aten.fill_.Scalar,
+        aten.fill_.Tensor,
+        aten.zero_.default,
+    }
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)